### PR TITLE
Multimodal VQA example for Captum Insights

### PR DESCRIPTION
**Pull request by miguelmartin75**
Added a multimodal VQA example for captum insights. Based off #131, will merge that prior to this.

### CODE REVIEW

File: captum/attr/_utils/common.py [hunk 1]
```diff
@@ -13,6 +13,14 @@ class ExpansionTypes(Enum):
     repeat_interleave = 2
 
 
+def safe_div(denom, quotient, default_value=None):
+    r"""
+        A simple utility function to perform `denom / quotient`
+        if the statement is undefined => result will be `default_value`
```
Comment (NarineK) [inline captum/attr/_utils/common.py]: After merging #131 this changes shouldn't show up here any more ...
You might need to do a rebase ...

File: tutorials/Multimodal_VQA_Captum_Insights.ipynb [hunk 2]
```diff
@@ -0,0 +1,553 @@
...
+    "\n",
+    "sys.path.append(PYTORCH_VQA_DIR)\n",
+    "sys.path.append(PYTORCH_RESNET_DIR)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, we will import the necessary modules to run the code in this tutorial. Please make sure you have the [prequesites to run captum](https://captum.ai/docs/getting_started), along with the pre-requisites to run this tutorial (as described in the first section)."
```
Comment (edward-io) [inline tutorials/Multimodal_VQA_Captum_Insights.ipynb]: nit: spelling. prerequisites

File: tutorials/Multimodal_VQA_Captum_Insights.ipynb [hunk 3]
```diff
@@ -0,0 +1,553 @@
...
+    "device = torch.device(\"cuda:0\" if torch.cuda.is_available() else \"cpu\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# VQA Model Setup\n",
+    "\n",
+    "Let's load the VQA model (again, please refer to the [model interpretatoin tutorial on VQA](https://captum.ai/tutorials/Multimodal_VQA_Interpret) if you want details)"
```
Comment (edward-io) [inline tutorials/Multimodal_VQA_Captum_Insights.ipynb]: nit: spelling. interpretation

File: tutorials/Multimodal_VQA_Captum_Insights.ipynb [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+    "        a = self.attention(v, q)\n",
+    "        v = apply_attention(v, a)\n",
+    "\n",
+    "        combined = torch.cat([v, q], dim=1)\n",
+    "        answer = self.classifier(combined)\n",
+    "        return answer\n",
+    "\n",
+    "vqa_resnet = VQA_Resnet_Model(vqa_net.module.text.embedding.num_embeddings)\n",
+    "\n",
+    "# `device_ids` contains a list of GPU ids which are used for paralelization supported by `DataParallel`\n",
```
Comment (edward-io) [inline tutorials/Multimodal_VQA_Captum_Insights.ipynb]: parallelization

File: tutorials/Multimodal_VQA_Captum_Insights.ipynb [hunk 5]
```diff
@@ -0,0 +1,553 @@
...
+    "    return interpretable_embedding.indices_to_embeddings(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using the Insights API\n",
+    "\n",
+    "Finally we have reached the relavent part of the tutorial.\n",
```
Comment (edward-io) [inline tutorials/Multimodal_VQA_Captum_Insights.ipynb]: *Relevant

File: tutorials/Multimodal_VQA_Captum_Insights.ipynb [hunk 6]
```diff
@@ -0,0 +1,553 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "features = [\n",
+    "    ImageFeature(\n",
+    "        \"Picture\",\n",
+    "        input_transforms=[lambda x: transform_normalize(x)],\n",
```
Comment (edward-io) [inline tutorials/Multimodal_VQA_Captum_Insights.ipynb]: I don't think you need a lambda here, you can just pass in transform_normalize to input_transforms directly.

**Comment by NarineK** [comment]
@miguelmartin75 , you need to land after import ...

**Pull request closed without merging**